src/relay/backend/contrib/ethosn/codegen_ethosn.h

```diff
@@ -240,18 +241,14 @@ struct EthosnCompilerConfigNode : public tvm::AttrsNode<EthosnCompilerConfigNode
   bool block_config_8x32;
   bool block_config_8x8;
   bool enable_intermediate_compression;
-#if _ETHOSN_API_VERSION_ == 2008
-  bool dump_debug_files;
-#endif
   bool disable_winograd;
   String debug_dir;
   String compiler_algorithm;
 
   TVM_DECLARE_ATTRS(EthosnCompilerConfigNode, "ext.attrs.EthosnCompilerConfigNode") {
-    TVM_ATTR_FIELD(variant)
-        .describe(
-            "0 for Ethos-N77, 1 for Ethos-N57, 2 for Ethos-N37,"
-            " 3 for Ethos-N78. See Ethos-N documentation.")
+    TVM_ATTR_FIELD(variant).describe("See Ethos-N documentation.").set_default("Ethos-N77");
+    TVM_ATTR_FIELD(sram_size_bytes)
+        .describe("Optionally override the default sram size. See Ethos-N documentation.")
```

Comment:
Is defaulting the sram size to 0 the correct thing to do?

Reply:
Yes, zero is used by the ethos-n driver stack to indicate that the default sram size is not to be overridden.